crates/driver/src/infra/config/file/mod.rs

```diff
@@ -192,6 +194,25 @@ struct SolverConfig {
     /// Determines whether the `solver` or the `driver` handles the fees
     #[serde(default)]
     fee_handler: FeeHandler,
+
+    /// List of addresses which receives a specific discount for the protocol
+    /// fees
+    #[serde(default)]
+    protocol_fee_discounted_addresses: Vec<ProtocolFeeDiscountedAddresses>,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+struct ProtocolFeeDiscountedAddresses {
+    address: H160,
+    kind: ProtocolFeeDiscountKind,
```

Comment:
Creating this `enum` as a placeholder to have in the future different kind of discounts and not only full one.